popup.js

```diff
@@ -12,12 +12,31 @@ function sendRequest(data) {
 }
 
 jQuery(document).ready(function($) {
-    //send message request to content script
-    $('#btnUpload').click(function() {
-        sendRequest({
-            type: 'addJson',
-            jsontext: $('#jsontext').val()
-        });
+    $('#form-import').submit(function(e) {
```

Comment:
@J2TeaM  I know but actually i mean, why we need `form` with `submit` instead of the click's event of `import` button, because I don't know your mean in this changes (i don't know why but I dont like submit from the past ^^ ).